tests/Unit/Pypp/PyppFundamentals.hpp

```diff
@@ -193,6 +197,32 @@ struct ToPyObject<T, Requires<tt::is_std_array_v<T>>> {
   }
 };
 
+template <>
+struct ToPyObject<DataVector, std::nullptr_t> {
+  static PyObject* convert(const DataVector& t) {
+    import_array();
+    long dims[1] = {static_cast<long>(t.size())};
```

Comment:
why not use std::array?

Reply:
Because constructing the PyArray requires a c-style array of the number of components in each dimension. I could make this a std::array but I'd only be doing so to call `.data()` on it anyway so I thought it'd be clearer to keep it as a c-style array since that what the function below takes. 

Is there a way to inline the construction of the c-style array in the call to PyArray_SimpleNew? Since it's only used there it might make it clearer.

Reply:
@nilsdeppe do you have an opinion?

Reply:
I am in favor of using `std::array`. Inside the H5 interface C-style arrays are "needed" and there I used `std::array` and called `.data()`. In general, https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-stack